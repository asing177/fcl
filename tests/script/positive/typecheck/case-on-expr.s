foo(int m, int n) {
  case (m + n) {
    1 -> "good";
    2 -> "ok";
    3 -> "meh";
    _ -> "blerg"
  }
}