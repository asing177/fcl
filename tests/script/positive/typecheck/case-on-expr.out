 
Method arguments and effects:
foo: (int, int) -> {write via terminate, write x}